homeassistant/components/air_pollutants/__init__.py

```diff
@@ -0,0 +1,172 @@
+"""
+Component for handling Air Pollutants data for your location.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/air_pollutants/
+"""
+import logging
+
+from homeassistant.helpers.entity_component import EntityComponent
+from homeassistant.helpers.config_validation import PLATFORM_SCHEMA  # noqa
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'air_pollutants'
+
+ENTITY_ID_FORMAT = DOMAIN + '.{}'
+
+ATTR_AIR_POLLUTANTS_AQI = 'air_quality_index'
+ATTR_AIR_POLLUTANTS_ATTRIBUTION = 'attribution'
+ATTR_AIR_POLLUTANTS_C02 = 'carbon_dioxide'
+ATTR_AIR_POLLUTANTS_CO = 'carbon_monoxide'
+ATTR_AIR_POLLUTANTS_N2O = 'nitrogen_oxide'
+ATTR_AIR_POLLUTANTS_NO = 'nitrogen_monoxide'
+ATTR_AIR_POLLUTANTS_NO2 = 'nitrogen_dioxide'
+ATTR_AIR_POLLUTANTS_OZONE = 'ozone'
+ATTR_AIR_POLLUTANTS_PM_0_1 = 'particulate_matter_0_1'
+ATTR_AIR_POLLUTANTS_PM_10 = 'particulate_matter_10'
+ATTR_AIR_POLLUTANTS_PM_2_5 = 'particulate_matter_2_5'
+ATTR_AIR_POLLUTANTS_SO2 = 'sulphur_dioxide'
+
+
+async def async_setup(hass, config):
+    """Set up the air pollutants component."""
+    component = hass.data[DOMAIN] = EntityComponent(_LOGGER, DOMAIN, hass)
```

Comment:
Should we set the default scan interval to 30 seconds as for sensors, instead of 15 seconds as is the main default.

Reply:
We should.